### ISSUE TITLE
model.evaluate() gives a different loss on training data from the one in training process

### ISSUE THREAD

**Issue by ydup** [cauchyturing/UCR_Time_Series_Classification_Deep_Learning_Baseline#17]
Hi, lately, I applied your baseline model - FCN with keras, but I found the similar problem with [this issue](https://github.com/keras-team/keras/issues/6977). I wonder if this will effect the performance.

Thanks.
